scipy/stats/_stats.pyx

```diff
@@ -0,0 +1,31 @@
+#cython: boundscheck=False
+#cython: nonecheck=False
```

Comment:
I think this is better done at a function level, especially in view of consolidating functions some of which may be user-facing.